templates/strings/objc-h.stencil

```diff
@@ -0,0 +1,60 @@
+// Generated using SwiftGen — https://github.com/SwiftGen/SwiftGen
+
+{% if tables.count > 0 %}
+@import Foundation;
+
+NS_ASSUME_NONNULL_BEGIN
+
+{% macro parametersBlock types %}{% filter removeNewlines:"leading" %}
+  {% for type in types %}
+    ({% call paramTranslate type %})p{{ forloop.counter }}{% if not forloop.last %} and:{% endif %}
+  {% endfor %}
+{% endfilter %}{% endmacro %}
+{% macro argumentsBlock types %}{% filter removeNewlines:"leading" %}
+  {% for type in types %}
+    p{{forloop.counter}}{% if not forloop.last %}, {% endif %}
+  {% endfor %}
+{% endfilter %}{% endmacro %}
+{% macro paramTranslate swiftType %}
+  {% if swiftType == "Float" %}
+    float
+  {% elif swiftType == "Int" %}
+    NSInteger
+  {% elif swiftType == "CChar" %}
+    char
+  {% elif swiftType == "String" %}
+    NSString*
+  {% elif swiftType == "UnsafePointer<CChar>" %}
+    char*
+  {% elif swiftType == "UnsafeRawPointer" %}
+    void*
+  {% else %}
+    objc-h.stencil is missing '{{swiftType}}'
+  {% endif %}
+{% endmacro %}
+{% macro recursiveBlock table item sp %}
+{% for string in item.strings %}
+{% if not param.noComments %}
+// {{string.key}} --> "{{string.translation}}"
```

Comment:
I don't think we have the `key` in the other templates, maybe remove it?

Reply:
I think it's essential.  Multiple keys may have the same (English) translation, so just knowing the translation isn't enough to find the string in `Localizable.strings`

Reply:
Right, but the key is in the function name (kinda), so this isn't really necessary.

Reply:
It's the "kinda" that makes me want to keep the actual key.  I'd rather not ask the user to know that to find the value returned by `+[Localizable.fooBar]`, they need to look for `foo.bar`.

Reply:
(Oh, BTW, process question: I 'resolved' the conversations to indicate I'd answered them, is that the right way to do things?)

Reply:
I usually resolve conversations when the outcome is clear (ok code changed). This was a bit more open-ended. I was going through the whole thing again anyway 🤷‍♂️